src/scripts/main.js

```diff
@@ -1,3 +1,72 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
```

Comment:
Code Quality Issue: It's a good practice to check if the element exists before using it. This can prevent potential runtime errors. Consider adding a check if the table and button elements are not null before using them.